docs/lib/docsify-mermaid-zoom.js

```
@@ -0,0 +1,96 @@
+                        e.addEventListener("mouseover", () => {
+                            (t.style.visibility = "visible"),
+                                (t.style.display = "flex"),
+                                (t.style.justifyContent = "flex-start");
+                        }),
+                        e.addEventListener("mouseleave", () => {
+                            t.style.visibility = "hidden";
+                        });
+                    const h = (b, k) => {
+                            const n = document.createElement("button");
+                            return (
+                                (n.innerHTML = k),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)